python/lsst/ts/externalscripts/base_make_calibrations.py

```diff
@@ -599,6 +605,186 @@ async def verify_calib(self, image_type, job_id_calib, exposure_ids_dict):
 
         return response
 
+    async def check_verification_stats(self, image_type, job_id_calib, job_id_verify):
+        """Check verification statistics.
+
+        Parameters
+        ----------
+        image_type:
+            Image type. Verification currently only implemented for ["BIAS",
+            "DARK", "FLAT"].
+
+        jod_id_calib : `str`
+            Job ID returned by OCPS during previous calibration
+            generation pipetask call.
+
+        job_idverify : `str`
+            Job ID returned by OCPS during previous calibration
+            verification pipetask call.
+
+        Returns
+        -------
+        certify_calib : `bool`
+            Booolean indicating if the calibration should be certified or not.
+
+        numStatErrors : `dict`[`str`][`str`]
+            Dictionary with the total number of tests failed per exposure and
+            per cp_verify test type. If there are not any tests that failed,
+            `None` will be returned.
+
+        thresholds : `dict`[`str`][`int`]
+            Dictionary reporting the different thresholds used to decide
+            whether a calibration should be certified or not (see `Notes`
+            below). If there are not any tests that failed,
+            `None` will be returned.
+
+        verify_stats : `dict`
+            Statistics from cp_verify.
+        """
+        # Collection name containing the verification outputs.
+        verifyCollection = f"u/ocps/{job_id_verify}"
```

Comment:
Mixing `camelCase` with `snake_case` in the code. I'd suggest `verify_collection`.

Reply:
Yes, sorry. The DM style guide was `camelCase` only until just recently (now there's an option, I think), but sometimes my muscle memory just does it like that and I forget that Telescope and Site is `snake_case` :)